docs/changes/constants/18118.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added CODATA 2022 support in `astropy.constants`, but did not make them default
+because the redefinition of SI units that will follow has not been implemented yet.
```

Comment:
For the benefit of end-users reading this change log, is it possible to list the affected SI units here or is the list too long?

Thanks!